cfgov/v1/__init__.py

```diff
@@ -58,10 +63,12 @@ def environment(**options):
         'get_protected_url': get_protected_url,
         'get_latest_activities': get_latest_activities,
         'get_snippets': get_snippets,
+        'localtime': template_localtime,
     })
 
     env.filters.update({
         'linebreaksbr': linebreaksbr,
+        'localtime': template_localtime,
```

Comment:
sorry assuming we have to register it, can we call it the same thing, e.g. `template_localtime: template_localtime`